packages/playwright/src/index.ts

```diff
@@ -249,8 +251,7 @@ const playwrightFixtures: Fixtures<TestFixtures, WorkerFixtures> = ({
     // Now that default test timeout is known, we can replace zero with an actual value.
     testInfo.setTimeout(testInfo.project.timeout);
 
-    const pageSnapshot = process.env.PLAYWRIGHT_COPY_PROMPT ? 'on' : 'off';
-    const artifactsRecorder = new ArtifactsRecorder(playwright, tracing().artifactsDir(), screenshot, pageSnapshot);
+    const artifactsRecorder = new ArtifactsRecorder(playwright, tracing().artifactsDir(), screenshot, !!process.env.PLAYWRIGHT_COPY_PROMPT);
```

Comment:
Let's opt-in by default, and opt-out with `process.env.PLAYWRIGHT_NO_COPY_PROMPT` instead.